x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/shared/ml_inference/utils.ts

```diff
@@ -58,6 +58,13 @@ export const isSupportedMLModel = (model: TrainedModelConfigResponse): boolean =
   );
 };
 
+export const getModelTobeFiltered = (model: TrainedModelConfigResponse): string | undefined => {
```

Comment:
nit: I think this function would be more clear if it were named `getTextExpansionModelId` or similar. I also think it make make more sense in `ml_inference_logic.ts` instead of in `utils.ts`, just because that's the only place it's used.